dynamic-plugins/imports/package.json

```diff
@@ -18,9 +18,9 @@
   "peerDependencies": {
     "@janus-idp/backstage-plugin-3scale-backend": "1.5.15",
     "@janus-idp/backstage-plugin-aap-backend": "1.6.15",
-    "@janus-idp/backstage-plugin-acr": "1.4.13",
+    "@janus-idp/backstage-plugin-acr": "1.6.4",
     "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.9",
-    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.11",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.10",
```

Comment:
should this be a new release, 1.4.13? why 1.4.10 when there are newer reeases going up to 1.7.9 and 1.8.5? 

https://www.npmjs.com/package/@janus-idp/backstage-plugin-jfrog-artifactory?activeTab=versions